.github/workflows/swift.yml

```diff
@@ -81,6 +92,17 @@ jobs:
     - uses: maxim-lobanov/setup-xcode@v1
       with:
         xcode-version: 14.1.0
+    - name: Install Homebrew
+      run: |
+        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+    - name: Install Hub
+      run: |
+        brew install hub
+
+    - name: Verify Hub Installation
+      run: |
+        hub version
```

Comment:
Can we put them together in a single step?

Reply:
Yes, we can. Also remove the `Verify Hub Installation` step to keep the flow precise.